lighthouse-core/gather/gather-runner.js

```diff
@@ -494,6 +494,39 @@ class GatherRunner {
     };
   }
 
+  /**
+   * Populates the important base artifacts from a fully loaded test page.
+   * Currently must be run before `start-url` gatherer so that `WebAppManifest`
+   * will be available to it.
+   * @param {LH.Gatherer.PassContext} passContext
+   * @return {LH.Artifacts.} passContext
+   */
+  static async getInstallabilityErrors(passContext) {
+    const response =
+      await passContext.driver.sendCommand('Page.getInstallabilityErrors');
+
+    let errors = response.installabilityErrors;
+    // Before M82, `getInstallabilityErrors` was not localized and just english
+    // error strings were returned. Convert the values we care about to the new error id format.
+    if (!errors) {
+      // @ts-ignore
```

Comment:
how about
```suggestion
      /** @type {Array<string>} */
      // @ts-ignore - intentionally support older incompatible protocol data
      const m81StyleErrors = response.errors || [];
      errors = m81StyleErrors.map(error => {
```

primarily care about narrowing the scopre of the tsignore

Reply:
nice